app/src/components/common/BalanceRow.vue

```diff
@@ -101,10 +113,14 @@ export default {
       type: Object,
       required: true,
     },
-    totalRewardsDenom: {
+    totalRewardsPerDenom: {
       type: Object,
       default: () => {},
     },
+    stake: {
```

Comment:
each balance should already have a property stating that is stake or not

Reply:
you mean in the object Balance coming from API itself? Then it is another PR I guess

Reply:
each `balanceV2` has a `type` stating if it is a currency or stake. but it is actually more versatile how it is.